lib/dry/web/roda/templates/Rakefile.tt

```diff
@@ -15,11 +15,30 @@ def db
   <%= config[:camel_cased_app_name] %>::Container["persistence.db"]
 end
 
+def settings
+  <%= config[:camel_cased_app_name] %>::Container["settings"]
+end
+
+def execute_database_create_command(settings)
```

Comment:
Since we're only using this in one place, I don't mind this being right inside the db:create task rather than in a method like this.